Pester.Tests.ps1

```diff
@@ -21,33 +24,9 @@ Describe -Tags 'VersionChecks' "Pester manifest and changelog" {
         $script:manifest.Guid | Should Be 'a699dea5-2c73-4616-a270-1f7abb777e71'
     }
 
-    It "has a valid version in the manifest" {
-        $script:manifest.Version -as [Version] | Should Not BeNullOrEmpty
-    }
-
-    $script:changelogVersion = $null
-    It "has a valid version in the changelog" {
-
-        foreach ($line in (Get-Content $changeLogPath))
-        {
-            if ($line -match "^\D*(?<Version>(\d+\.){1,3}\d+)")
-            {
-                $script:changelogVersion = $matches.Version
-                break
-            }
-        }
-        $script:changelogVersion                | Should Not BeNullOrEmpty
-        $script:changelogVersion -as [Version]  | Should Not BeNullOrEmpty
-    }
-
-    It "changelog and manifest versions are the same" {
-        $script:changelogVersion -as [Version] | Should be ( $script:manifest.Version -as [Version] )
-    }
-
-    if (Get-Command git.exe -ErrorAction SilentlyContinue)
-    {
+    if (Get-Command git.exe -ErrorAction SilentlyContinue) {
```

Comment:
Yeah, the next one command to fix to achieve compatibility with OSes other than Windows :-/